configs/_base_/datasets/waymoD5-mv3d-3class.py

```diff
@@ -19,7 +19,7 @@
 #      }))
 backend_args = None
 
-class_names = ['Car', 'Pedestrian', 'Cyclist']
+class_names = ['Pedestrian', 'Cyclist', 'Car']
```

Comment:
seems to be a change break compatibility. Does this modification fix previous bug?

Reply:
It's to align with the Depth-from-Motion repo. This config has never been used correctly before. It will not break compatibility.